src/OSPSuite.UI/Extensions/ChartControlExtensions.cs

```diff
@@ -84,6 +84,14 @@ private static void setFontAndSizeSettings(ChartControl chartControl, int width,
 
       public static void CopyToClipboard(this UxChartControl chartControl, string watermark)
       {
+         if (string.IsNullOrEmpty(watermark))
+         {
+            //Bug in devexpress that does not keep the color in bar charts
+            chartControl.CopyChartToClipboard(chartControl);
```

Comment:
This is the change for the copy issue. Actually the reason is that the clone method does not copy the event handlers. Colors are set with event handlers for bars so this is the reason. The fix is to iterate over all series and copy the color. Too risky for now => e.g. draft plot may have wrong colors :)